lib/dependency-versions.ts

```diff
@@ -342,13 +345,15 @@ export function fixMismatchingVersions(
         package_.packageJson.devDependencies[mismatchingVersion.dependency] !==
           fixedVersion
       ) {
-        writeDependencyVersion(
-          package_.pathPackageJson,
-          package_.packageJsonEndsInNewline,
-          'devDependencies',
-          mismatchingVersion.dependency,
-          fixedVersion
-        );
+        if (!dryrun) {
+          writeDependencyVersion(
+            package_.pathPackageJson,
```

Comment:
Is there a reason `package_` is scattered throughout this file instead of a more descriptive name?

Reply:
`package_` just refers to an individual package. `package` cannot be used since it's a reserved keyword in JavaScript.